src/Console/Application.php

```diff
@@ -193,10 +194,16 @@ private function parseConfiguration(InputInterface $input)
             $this->config['adapter'] = 'web';
             $this->config['webPath'] = $input->getParameterOption('--web-path');
             $this->config['webUrl'] = $input->getParameterOption('--web-url');
+            if($input->hasParameterOption('--web-allow-insecure')) {
```

Comment:
Instead of setting `config['httpclient']` use `config['webAllowInsecure']` instead and when constructing the HTTP create the array with verify_peer and verify_host if `webAllowInsecure` is set to true.